src/golang/lib/job/process.go

```diff
@@ -291,18 +291,18 @@ func (j *ProcessJobManager) Config() Config {
 }
 
 func (j *ProcessJobManager) Launch(
-	ctx context.Context,
+	_ context.Context,
```

Comment:
nit: should we remove this param if its unused

Reply:
I think it's used by the other job manager implementations though.